### PR TITLE
Personal page shahinsha.

### PR DESCRIPTION
**Pull request by muhammedshahinshapottayil**
I am muhammed shahinsha p . This is my personal page pull request please review it. and let me know if any changes to be made or approve this.

Your ENS/address: muhammedshahinshap.eth / 0xc26816404F551a7b3964B0D665380e9e3aca17b3

### CODE REVIEW

**Review by technophile-04 — COMMENTED** [review]
Hey thanks @muhammedshahinshapottayil, could you please revert [5de84d8](https://github.com/BuidlGuidl/batch3.buidlguidl.com/pull/13/commits/5de84d8c7cd8b088bd4a4db771e3f94276210616), Lets handle it in separate PR 🙌

File: packages/nextjs/app/builders/0xc26816404F551a7b3964B0D665380e9e3aca17b3/svgs.tsx
[the remark this answers is not in the document]
Comment (muhammedshahinshapottayil) [reply packages/nextjs/app/builders/0xc26816404F551a7b3964B0D665380e9e3aca17b3/svgs.tsx:1]: Completed

**Comment by muhammedshahinshapottayil** [comment]
Reverted commit 5de84d8 on the last commit now only personal page file changes are included in this PR.

**Review by technophile-04 — APPROVED** [review]
Hey thanks @muhammedshahinshapottayil, just pushed 
21bd107 to make it more Intuitive and others can use it too 🙌

**Pull request merged**